packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderContext/DataConfigs.js

```diff
@@ -106,7 +112,16 @@ export const valueRenderers = {
     return config.unit && val ? `${val} ${config.unit}` : val;
   },
   option: (value) => {
-    return Array.isArray(value) ? value.join(', ') : value;
+    if (value && typeof value == 'string') {
+      return value;
+    } else if (value) {
+      let selectedValues = value && Array.isArray(value) ? value : [value];
```

Comment:
Looks like another case where you don't need to be using `let` as the value of `selectedValues` is never re-assigned.

Reply:
modified as advised